### DISCUSSION TITLE
How do I create a new page?

### DISCUSSION THREAD

**Discussion by JohanneBrams** [vercel/next.js#23654] [Help]
I followed the guidelines for creating a new page, but every time I try to run it, it just says ''404: This page could not be found''. 
I have saved the file in a map ''posts'' inside the map ''pages'' so what am I doing wrong?

**Comment by JonathanCramer** [comment]
When you say "map", do you mean "folder"?
If so, the new page should be found under "localhost:3000/posts/nameofpage".

If you want it to be found under the posts path without the "nameofpage", you should store the nameofpage.js file as posts.js

**Comment by JohanneBrams** [comment]
Hey! Yes, when I said map I meant folder. 
But every time I try to run l http://localhost:3000 or  http://localhost:3000/post/first-posts.js it keeps saying that my server can't connect to the site? I have tried at home on my wifi and on a cafe but nothing can get it to open the website?

**Reply by JohanneBrams** [reply]
Hello again, I got the website to work. But do you know what I am doing wrong here? It says that something is wrong in my head section, but all I did was following the guide? 
![Skærmbillede 2021-04-05 kl  16 32 11](https://user-images.githubusercontent.com/81842361/113549619-aec7f700-962c-11eb-8a4a-e1e04af360d6.png)

**Reply by JonathanCramer** [reply]
About the head being undefined - that is because you need to import it...
Just type `import Head from 'next/head'`
